contents/teams/surveys/objectives.mdx

```diff
@@ -1,37 +1,62 @@
-# Goal 1: Talk to customers
+# ✅ Goal 1: Mobile surveys, Everywhere
 
-We'll only succeed if we are close to our customers, let's make our relationship with them stronger.
+We’ve already got mobile surveys working in React Native and iOS, and we’ve kicked off work for Flutter and Android [tracking here](https://github.com/PostHog/posthog/issues/25085). Next step: bring everything to feature parity with Web.
 
-- Do 5 customer interviews
-- Find the top 5 reference customers and build a relationship with them
-- Do a case study with a top reference customer
+- ✅ iOS
+- ✅ React Native
+- 🛠️ Android
+- 🛠️ Flutter
 
-# Goal 2: Improve quality
+# 📈 Goal 2: Level up survey analysis
 
-There are a couple of reported bugs that were not addressed quickly, we'd like to speed up this process and fix them.
+You asked — we listened. We're making survey analysis more useful and enjoyable, starting with a brand new global analytics page.
 
-- Reduce [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) to fix bugs from months to weeks
-- Reduce [paper cut](https://en.wikipedia.org/wiki/Lingchi) issues by 50%
-    - Currently around ~30 [open bugs](https://github.com/PostHog/posthog/issues?q=is%3Aopen+is%3Aissue+label%3Afeature%2Fsurveys+label%3Abug)
+- [Global analytics view](https://github.com/PostHog/posthog/issues/28418)
+- A cleaner, more useful survey responses page
+- Extra polish and UX improvements across the board
 
-# Goal 3: Finish up [WIP](https://en.wikipedia.org/wiki/Work_in_process) features
+# 🎨 Goal 3: Survey customization
 
-We've started and nearly finished a couple of features our customers requested, but they are not [GA](https://en.wikipedia.org/wiki/Software_release_life_cycle#General_availability_(GA)) yet, the goal is to finish them up, gather feedback and put them in front of customers.
+A lot of people use the API to tweak their surveys. Let’s bring those use cases into the product.
 
-- [Adaptive Sampling](https://github.com/PostHog/posthog/issues/25722)
-    - Currently under a feature flag
-- [Partial Responses](https://github.com/PostHog/posthog/issues/19300)
-    - There are Draft PRs
-- [Survey Actions](https://github.com/PostHog/posthog/issues/27378)
-    - Currently under a feature flag
-- [AI Summaries](https://github.com/PostHog/posthog/issues/27379)
-    - Currently under early access features
+- Better wait period controls
+- [Alignment customization](https://github.com/PostHog/posthog/issues/28479)
+- [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
+- A handful of other power-user options
```

Comment:
Maybe instead of this option (feels vague) we can add:

- Allow adding icons or HTML to multiple/single choice options (key customers ask)
- Allow displaying more than one question at a time and increasing the popover size (key customers ask)
- Allow "one click response" - click on a choice, survey is responded straight away (key customers ask)
- Allow embedding person properties in link surveys (key customers ask)

issues to be created

Reply:
removed A handful of other power-user options
and added more links